### PR TITLE
generation dispatch : use frequency reserve

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

**Review by ayolab — APPROVED** [review]
Tests OK
Code OK

File: src/translations/en.json [hunk 1]
```diff
@@ -925,6 +925,9 @@
     "DefaultOutageRate": "Default outage rate",
     "GeneratorsWithoutOutage": "Generators without outage simulation",
     "GeneratorsWithFixedActivePower": "Generators with fixed active power",
+    "GeneratorsFilters": "GENERATOR FILTER",
```
Comment (Walid-Sahnoun) [inline src/translations/en.json:928]: should change "GeneratorsFilters" to "GeneratorFilter" ? (remove the "s")

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
tests ok

File: src/translations/en.json (hunk 1 above)
Comment (FranckLecuyer) [reply src/translations/en.json]: Renamed

File: src/components/utils/rhf-inputs/directory-items-input.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ const DirectoryItemsInput = ({
     equipmentTypes, // Mostly used for filters, it allows the user to get elements of specific equipment only
     itemFilter, // Used to further filter the results displayed according to specific requirement
     titleId, // title of directory item selector dialogue
+    disableErrorMessage,
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/components/utils/rhf-inputs/directory-items-input.js]: Renamed

**Pull request merged**